src/packages/@app/pages/Dashboard/Dashboard.ViewModel.ts

```diff
@@ -362,6 +364,10 @@ export class DashboardViewModel {
       await this.workspaceRepository.setActiveWorkspace(res._id);
       notifications.success("New Workspace Created");
     }
+    MixpanelEvent(Events.Create_New_Workspace_TopBar, {
+      source: "create new workspace from top bar",
```

Comment:
we don't give the event name in source, source is generally referred to the place from where the event has been triggered.